tldw_Server_API/app/core/Explainer/service.py

```
@@ -0,0 +1,271 @@
+                intent=effective_intent,
+            )
+        except ExplainerTerminalJobError as exc:
+            raise ExplainerValidationError(str(exc)) from exc
+        queued = self.repo.update_node(
+            session_id,
+            node_id,
+            owner_user_id=owner_user_id,
+            status=ExplainerNodeStatus.QUEUED.value,
+        )
+        if queued is None:
+            raise ExplainerNotFoundError("Explainer node not found")
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)